assets/templates/kubernetes.container/schema-b/configs.yml

```diff
@@ -45,7 +45,7 @@ fields:
   - name: container.name
     enum: ["web", "default-http-backend", "dnsmasq", "csi-driver", "web", "web", "web", "prometheus", "konnectivity-agent", "sidecar", "kubedns", "metrics-server-nanny", "web", "web", "fluentbit", "autoscaler", "gke-metrics-agent", "elastic-agent", "web", "kube-state-metrics", "metrics-server", "fluentbit", "elastic-agent", "web", "prometheus-to-sd-exporter"]
   - name: timedate
-    # Add more dates below in case you want your data to be spread
+    # Add more dates below in case you want your data to be spread.Those are fixed because this is required in the nightly tests of Rally
```

Comment:
can you explain about this?
will it be fixed "forever", or only for the current nightly tests of rally.
what happens if we use `timestamp` with `period: 2880m` (two days) like I did [here](https://github.com/elastic/integrations/pull/8444)

Reply:
This one is fixed for the current nightly tests of rally. We have specific visualisations and we want our data to be in specific range in order our queries to be be always specific

Reply:
would it be possible to change the visualisation range to be `-2d/now`?

Reply:
@aspacca 
I have made some tests today and from generator tool side we can change the timedate to date type with [period: 48h] and all seem ok. So I dont see an issue with this now

Moreover on the rally side we have hardcoded the values of visualisations [here](https://github.com/elastic/rally-tracks/pull/513)

So we need to find a way to import to calculate the new itervals and end_time based on the track provided. 
cc @martijnvg for visibility